src/preload/links.js

```diff
@@ -11,6 +13,13 @@ const handleAnchorClick = (event) => {
 	const href = a.getAttribute('href');
 	const download = a.hasAttribute('download');
 
+	const isPdfFile = /.*\.pdf$/.test(href);
+	if (isPdfFile) {
+		// todo need to prevent the download file window from appearing
```

Comment:
In the previous version, the electron system download window was not shown, but now system download window is shown and then window with pdf shown. Perhaps this is due to some changes in the new version? It is necessary that only one window is displayed, a window with pdf